### PR TITLE
tests: moving interfaces tests to snaps-state tool

### PR DESCRIPTION
**Pull request by sergiocazzolato**
Part 3 of the full migration

### CODE REVIEW

File: tests/main/interfaces-network-control/task.yaml [hunk 1]
```diff
@@ -20,12 +20,8 @@ environment:
     ARP_ENTRY_ADDR: "30.30.30.30"
 
 prepare: |
-    #shellcheck source=tests/lib/snaps.sh
-    . "$TESTSLIB/snaps.sh"
-    #shellcheck source=tests/lib/systemd.sh
-    . "$TESTSLIB/systemd.sh"
```
Comment (bboozzoo) [inline tests/main/interfaces-network-control/task.yaml:26]: systemd was unused here?

**Review by bboozzoo — APPROVED** [review]
LGTM

File: tests/main/interfaces-network-control/task.yaml (hunk 1 above)
Comment (stolowski) [reply tests/main/interfaces-network-control/task.yaml:26]: It seems so, but let's have it confirmed by Sergio.

**Review by stolowski — APPROVED** [review]
Thanks! +1

File: tests/main/interfaces-network-control/task.yaml (hunk 1 above)
Comment (zyga) [reply tests/main/interfaces-network-control/task.yaml:26]: Yes, it is unused, you can expand this section just a little below to see.

**Review by zyga — APPROVED** [review]
LGTM

**Pull request merged**